Insights feature critique/Insights Feature Critique.dc.html

```
@@ -0,0 +1,895 @@
+          <p style="margin:0;font-size:14px;font-weight:500"><span style="font-family:var(--font-mono);font-size:12px;color:var(--text-subdued);margin-right:8px">A1</span>Cards nested inside cards on the dashboard</p>
+          <p style="margin:6px 0 0 0;font-size:13px;line-height:1.6;color:var(--text-secondary);max-width:760px">The dashboard section shell is already a card, and <span style="font-family:var(--font-mono);font-size:12px">_insights_feed</span> renders full insight cards inside it — white on white, ring on ring. The DS is explicit that cards sit directly on the surface and are never nested. Every other widget puts flush content (tables, charts, stat rows) inside the shell; Insights is the only one that re-wraps. See mockup <a href="#m2">M2</a>.</p>
+          <sc-if value="{{ showEvidence }}" hint-placeholder-val="{{ true }}"><p style="margin:6px 0 0 0;font-family:var(--font-mono);font-size:11px;line-height:1.6;color:var(--text-subdued)">dashboard.html.erb section = bg-container rounded-xl shadow-border-xs → wraps _insight_card = bg-container rounded-xl shadow-border-xs p-4</p></sc-if>
+        </div>
+      </div>
+
+      <div style="display:grid;grid-template-columns:96px 1fr;gap:16px;padding:16px 24px;border-top:1px solid var(--border-divider)">
+        <span style="align-self:start;justify-self:start;display:inline-flex;align-items:center;gap:5px;padding:2px 9px;border-radius:9999px;background:var(--color-red-50);color:var(--color-red-700);font-size:11px;font-weight:500"><span style="width:5px;height:5px;border-radius:9999px;background:var(--color-red-500)"></span>High</span>
+        <div>
+          <p style="margin:0;font-size:14px;font-weight:500"><span style="font-family:var(--font-mono);font-size:12px;color:var(--text-subdued);margin-right:8px">A2</span>Priority is doing sentiment's job — good news renders red</p>
+          <p style="margin:6px 0 0 0;font-size:13px;line-height:1.6;color:var(--text-secondary);max-width:760px">Icon color is mapped from priority (high → destructive, medium → warning) with a two-type allowlist for green. But urgency and sentiment are orthogonal, and the mapping breaks on real rows: a savings-rate <em style="font-style:normal;font-weight:500">improvement</em> of ≥10pp is priority high → a red piggy bank on good news; spending trending <em style="font-style:normal;font-weight:500">down</em> ≥50% is likewise red. The DS also reserves red for destructive UI — even genuinely negative amounts aren't red in Sure. Sentiment should pick the color; priority should pick the order.</p>
+          <sc-if value="{{ showEvidence }}" hint-placeholder-val="{{ true }}"><p style="margin:6px 0 0 0;font-family:var(--font-mono);font-size:11px;line-height:1.6;color:var(--text-subdued)">insights_helper.rb: POSITIVE_TYPES = [net_worth_milestone, budget_on_track] · savings_rate_change: delta.abs >= 10 → "high" regardless of direction</p></sc-if>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)